root/static/scripts/common/utility/entityLink.js

```diff
@@ -4,7 +4,7 @@
 // http://www.gnu.org/licenses/gpl-2.0.txt
 
 import React from 'react';
-import {l} from '../../common/i18n';
+import {l} from '../i18n';
```

Comment:
They actually both resolve to the same path, I'm not sure why it's moving out of ./common and then back again, currently.

I'll put this in a separate commit